stable/node-local-dns/values.yaml

```
@@ -33,6 +33,9 @@ serviceAccount:
 
 podAnnotations: {}
 
+securityContext:
+  privileged: true
```

It would probably be better to have this default to `false`, no?

```suggestion
  privileged: false
```